command/get_transitions.go

```diff
@@ -0,0 +1,73 @@
+package command
+
+import (
+	"encoding/json"
+	"github.com/ExpediaGroup/flyte-jira/client"
+	"github.com/HotelsDotCom/flyte-client/flyte"
+	"log"
+)
+
+var (
+	GetTransitions = flyte.Command{
+		Name:         "GetTransitions",
+		OutputEvents: []flyte.EventDef{getTransitionsEventDef, getTransitionsFailureEventDef},
+		Handler:      getTransitionsHandler,
+	}
+
+	getTransitionsEventDef = flyte.EventDef{
+		Name: "GetTransitions",
+	}
+
+	getTransitionsFailureEventDef = flyte.EventDef{
+		Name: "GetTransitionsFailure",
+	}
+)
+
+type (
+	transitionRequest struct {
+		IssueId string `json:"issueId"`
+	}
+
+	transitionsSuccessPayload struct {
+		Id          string `json:"id"`
+		results		[]client.Transition `json:"transitions"`
```

Comment:
I'd run this file through the formatter. (`goimports` preferred so we can get those imports arranged nicely too!)

Reply:
Great catch!